### DISCUSSION TITLE
Question on Framework Tagging and Course Editor Capabilities

### DISCUSSION THREAD

**Discussion by rajnishdargan** [Sunbird-Knowlg/Community#234] [Q&A]
Hello everyone,

I am exploring a scenario and would like your insights on its feasibility and implementation:

**While creating content:**
I would like to tag the content using **Framework1** categories:

- Learning Category
- Learning Sub Category

**While creating a course:**

The course should be tagged using **Framework2** categories:

- Board
- Medium
- Grade
- Subject

In the "Add from Library" functionality:

I want to search and add contents to the course based on **Framework1** tags (Learning Category, Learning Sub Category).

**My main question:**

Is the Course Editor capable of handling this scenario, where contents and courses are tagged with different frameworks, and the search functionality in "Add from Library" allows filtering based on Framework1 tags?
Any insights, suggestions, or examples of similar use cases would be greatly appreciated!

Thank you in advance for your help.
@pallakartheekreddy @HarishGangula
@coolbung @itsvick @snehal0904 @siddhi0719 @komalm @vinod303

**Comment by pallakartheekreddy** [comment]
@rajnishdargan Yes, `Add from Library` will show all the contents if you don't select any filters.
